tests/parser/run-lexer-tests.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
Could this use PHPUnit?

Reply:
Aha! This seems to be producing a stats report. Still, we can probably get the same from PHPUnit.